draft-ietf-rtcweb-jsep.xml

```diff
@@ -1609,6 +1609,30 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
             section below.
           </t>
         </section>
+        <section title="direction" anchor="sec.transceiver-direction">
+          <t>
+            The direction method returns the last value passed into
+            setDirection. If setDirection has never been called, it
+            returns the direction the transceiver was initialized with.
+          </t>
```

Comment:
If you initialize a transceiver with "sendrecv" and the answer is "recvonly", `direction` will be "sendrecv" and `currentDirection` will be "sendonly". Isn't this what we want?

I thought the last point was captured by the fact that `currentDirection` depends on the last applied answer for the transceiver.